lib/rubocop/cop/style/super_arguments.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for redundant argument forwarding when calling super
+      # with arguments identical to the method definition.
+      #
+      # @example
+      #   # bad
+      #   def method(*args, **kwargs)
+      #     super(*args, **kwargs)
+      #   end
+      #
+      #   # good - implicitly passing all arguments
+      #   def method(*args, **kwargs)
+      #     super
+      #   end
+      #
+      #   # good - forwarding a subset of the arguments
+      #   def method(*args, **kwargs)
+      #     super(*args)
+      #   end
+      #
+      #   # good - forwarding no arguments
+      #   def method(*args, **kwargs)
+      #     super()
+      #   end
+      class SuperArguments < Base
+        extend AutoCorrector
+
+        DEF_TYPES = %i[def defs].freeze
+
+        MSG = 'Call `super` without arguments and parentheses when the signature is identical.'
+
+        def on_super(super_node)
+          def_node = super_node.ancestors.find do |node|
+            # You can't implicitly call super when dynamically defining methods
+            break if define_method?(node)
+
+            break node if DEF_TYPES.include?(node.type)
+          end
+          return unless def_node
+          return unless arguments_identical?(def_node.arguments.argument_list, super_node.arguments)
+
+          add_offense(super_node) { |corrector| corrector.replace(super_node, 'super') }
+        end
+
+        private
+
+        # rubocop:disable Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Can you tweak to the following?

```suggestion
        # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```